src/04-modality-specific-files/11-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,331 @@
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| NIRSCWAMPLITUDE             | Continuous wave amplitude measurements. Equivalent to dataType 001 in SNIRF.                                                                                                 |
+| NIRSCWFLUORESCENSEAMPLITUDE | Continuous wave fluorescence amplitude measurements. Equivalent to dataType 051 in SNIRF.                                                                                    |
+| NIRSCWOPTICALDENSITY        | Continuous wave change in optical density measurements. Equivalent to dataTypeLabel dOD in SNIRF.                                                                            |
+| NIRSCWHBO                   | Continuous wave oxygenated hemoglobin (oxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbO in SNIRF.                                                  |
+| NIRSCWHBR                   | Continuous wave deoxygenated hemoglobin (deoxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbR in SNIRF.                                              |
+| NIRSCWMUA                   | Continuous wave optical absorption  measurements. Equivalent to dataTypeLabel mua in SNIRF.                                                                                  |
+| ACCEL                       | Accelerometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z). |
+| GYRO                        | Gyrometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z).     |
+| MAGN                        | Magnetomenter channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z). |
+| MISC                        | Miscellaneous                                                                                                                                                                |
+
+Example:
+
+```Text
+Name         type                   source      detector      wavelength_nominal   units
+S1-D1        NIRSCWAMPLITUDE        A1          Fz            760                  V
+S1-D1        NIRSCWAMPLITUDE        A1          Fz            850                  V
+S1-D2        NIRSCWAMPLITUDE        A1          Cz            760                  V
+S2-D1        NIRSCWAMPLITUDE        A2          Fz            760                  V
+S3-D4        NIRSCWAMPLITUDE        VisS2       VisD4         760                  V
+```
+
+## Optode description (`*_optodes.tsv`)
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["optodes"]) }}
```

Comment:
This filename template allows for `*_optodes.json`

We don't have rules for those in the validator. Is it a valid filename or included by mistake? Didn't see any examples of them in the two bids-examples PR.

Reply:
Don't all TSV files need to support sidecar files for defining column metadata? The rules for optodes files say `additional_columns: allowed_if_defined`, so there needs to be some place where those additional columns may be defined.